lib/provider/camunda/parts/VariableMappingProps.js

```diff
@@ -233,7 +233,12 @@ module.exports = function(group, element, bpmnFactory, translate) {
   group.entries.push(entryFactory.selectBox({
     id: 'variableMapping-inOutType',
     label: translate('Type'),
-    selectOptions: inOutTypeOptions,
+    selectOptions: inOutTypeOptions.map(function(option) {
```

Comment:
Options should be translated as soon as they appear in code as strings. Putting translate function in map will make it difficult actually translate applications.

Reply:
Agreed.
Done.